pytorch_lightning/core/lightning.py

```diff
@@ -542,7 +539,7 @@ def configure_ddp(self, model, device_ids):
         3. On a testing batch, the call goes to model.test_step
 
         Args:
-            model (LightningModule): the LightningModule currently being optimized
+            model (pytorch_lightning.core.LightningModule): the LightningModule currently being optimized
```

Comment:
add ` around the class

Reply:
Resolved. 4287eb0d5bf3da3a499aab0c275bc474892f3bc9